block_side_bar.php

```diff
@@ -277,8 +284,11 @@ public function get_content() {
      * location (site, course, blog, etc)
      */
     public function specialization() {
+        global $OUTPUT;
+
         if (!empty($this->config->title)) {
-            $this->title = $this->config->title;
+            $this->title = '<span><img src="'.$OUTPUT->image_url('monologo', 'hsuforum').
+                '" class="icon" alt="" /></span>'." ".$this->config->title;
```

Comment:
Reason for ." ".? Are you not able to add the space in the string or just use variable/string interpolation?